Splitio.Redis/Services/Cache/Classes/RedisEventsCache.cs

```diff
@@ -19,8 +20,8 @@ public class RedisEventsCache : RedisCacheBase, ISimpleCache<WrappedEvent>
         public RedisEventsCache(IRedisAdapterProducer redisAdapter, 
             string machineName,
             string machineIP, 
-            string sdkVersion, 
-            string userPrefix = null) : base(userPrefix) 
+            string sdkVersion,
+            RedisConfig redisConfig = null, bool clusterMode = false) : base(redisConfig, clusterMode) 
```

Comment:
you can remove `machineName`, `machineIP` and `sdkVersion` and you could use `redisConfig`